package.json

```diff
@@ -60,31 +60,29 @@
   ],
   "dependencies": {
     "@typescript-eslint/parser": "^4.0.0",
-    "eslint-config-standard": "^14.1.1"
+    "eslint-config-standard": "^16.0.0"
   },
   "peerDependencies": {
-    "eslint": ">=6.2.2",
-    "eslint-plugin-import": ">=2.18.0",
-    "eslint-plugin-node": ">=9.1.0",
-    "eslint-plugin-promise": ">=4.2.1",
-    "eslint-plugin-standard": ">=4.0.0",
-    "typescript": ">=3.9",
-    "@typescript-eslint/eslint-plugin": ">=4.0.1"
+    "eslint": "^7.12.1",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^4.2.1",
+    "typescript": "^3.9",
+    "@typescript-eslint/eslint-plugin": "^4.12.0"
   },
   "devDependencies": {
     "@commitlint/cli": "11.0.0",
     "@commitlint/travis-cli": "11.0.0",
     "@types/eslint": "7.2.6",
     "@types/node": "14.14.20",
-    "@typescript-eslint/eslint-plugin": "4.0.1",
+    "@typescript-eslint/eslint-plugin": "4.12.0",
```

Comment:
Is this related?

Reply:
I think that was from renovate; I just aligned dev & peer deps.
Should I revert both to 4.0.1?

Reply:
I've reverted this change.